tyk-single-dc/values.yaml

```diff
@@ -5,12 +5,22 @@
 global:
   license:
     dashboard: ""
+
+  adminUser:
+    firstName: admin
+    lastName: user
+    email: default@example.com
+    # Set a password or a random one will be assigned.
+    password: "123456"
   components:
     # Determines whether or not the pump component should be installed.
     pump: true
-
+    enterprisePortal: true
```

Comment:
will the enterprise portal be enabled by default? just asking.

Reply:
I set it to false and we'll discuss in next meeting.